src/components/multi-accordion-with-title/MultiAccordionWithTitle.styles.ts

```diff
@@ -7,18 +7,23 @@ const borderStyles = {
     borderRadius: '6px'
 }
 
+const accordionTitleStyles = (isMobile: boolean) => ({

```

Comment:
I don't see where this is used, is it needed at all?

Reply:
already removed